app/api/events.py

```diff
@@ -70,6 +70,13 @@ def validate_event(user, modules, data):
         raise ForbiddenException({'source': ''},
                                  "Only verified accounts can publish events")
 
+    if data.get('state', None) == 'published':
+        event = Event.query.filter_by(name=data.get('name', None)).first()
+        ticket_available = event.tickets_available
+        if (ticket_available == 0):
+            raise ForbiddenException({'source': ''},
```

Comment:
Forbidden is wrong error, it's mainly used for authorization errors. And point source correctly

Reply:
@iamareebjamal Please Check now